libraries/AP_Follow/AP_Follow.h

```diff
@@ -153,10 +165,12 @@ class AP_Follow
     AP_Int8     _alt_type;          // altitude source for follow mode
```

Comment:
Maybe change this to `AP_Enum<Location::AltType>` - it should clean up a bunch of code.